### PR TITLE
Post Result pipeline changes

### PR DESCRIPTION
**Pull request by anrao19**
# Description
Post Result pipeline changes to get mails for pass and fail

Screenshot from 2023-02-17 10-03-45

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by tintumathew10 — COMMENTED** [review]
LGTM

File: pipeline/post-results.groovy [hunk 1]
```diff
@@ -326,4 +335,4 @@ node("rhel-8-medium || ceph-qe-ci") {
         subject += "\n Jenkins URL: ${env.BUILD_URL}"
         googlechatnotification(url: "id:rhcephCIGChatRoom", message: subject)
     }
-}
```
Comment (manasagowri) [inline pipeline/post-results.groovy:329]: Could you please resolve this warning?

**Review by manasagowri — COMMENTED** [review]
Just a small change

File: pipeline/post-results.groovy (hunk 1 above)
Comment (anrao19) [reply pipeline/post-results.groovy]: Made changes

**Review by manasagowri — APPROVED** [review]
LGTM

**Pull request merged**